src/components/AppSidebar/AppSidebar.vue

```diff
@@ -199,8 +205,13 @@ export default {
 			default: '',
 			required: true,
 		},
+
+		/**
+		 * Allow to edit the sidebar title.
+		 * Accepts true, false, and 'toggle' for showing the input on click.
+		 */
 		titleEditable: {
-			type: Boolean,
+			type: [Boolean, String],
```

Comment:
Do we really want to have this as an option? I'm fine having the click to edit as default if `titleEditable` is true :)

Reply:
I don't mind. I just didn't want to change the current behavior to much, because I wasn't sure if it is needed like that somewhere.